spec/core_functions/selector/unify/compound.hrx

```diff
@@ -20,15 +20,37 @@ a {
 
 <===>
 ================================================================================
-<===> full_overlap/input.scss
+<===> full_overlap/class/input.scss
 @use "sass:selector";
 a {b: selector.unify(".c.d", ".c.d")}
 
-<===> full_overlap/output.css
+<===> full_overlap/class/output.css
 a {
   b: .c.d;
 }
 
+<===>
+================================================================================
+<===> full_overlap/pseudo_element/class_after/input.scss
+@use "sass:selector";
+a {b: selector.unify("::c.d", "::c.d")}
+
+<===> full_overlap/pseudo_element/class_after/output.css
+a {
+  b: ::c.d;
+}
+
+<===>
+================================================================================
+<===> full_overlap/pseudo/different_order/input.scss
+@use "sass:selector";
+a {b: selector.unify("::c.d", ".d::c")}
+
+<===> full_overlap/pseudo/different_order/output.css
+a {
+  b: ::c.d;
+}
```

Comment:
Both of these involve meaningless input selectors, so are probably not worth testing. Unifying `.c::d` with itself is worth testing, though, as well as cases with pseudo-selectors